### ISSUE TITLE
[breakpad] build failure, "Cannot open include file: 'atlbase.h': No such file or directory"

### ISSUE THREAD

**Issue by ghesketh** [microsoft/vcpkg#29531]
Package: breakpad[core]:x64-windows-static-md -> 2022-07-12#2

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.34.31937.0
-    vcpkg-tool version: 2022-12-14-7ae0d8527fb488fde10a89c2813802dc9b03b6f9
    vcpkg-scripts version: 185a7aa2 2023-02-06 (2 days ago)

**To Reproduce**

`vcpkg install breakpad`

**Failure logs**

```
-- Using cached google-breakpad-v2022.07.12.tar.gz.
-- Cleaning sources at C:/Users/Public/src/vcpkg.microsoft/buildtrees/breakpad/src/2022.07.12-b097ac1ad8.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/Users/Public/src/vcpkg.microsoft/downloads/google-breakpad-v2022.07.12.tar.gz
-- Applying patch fix-unique_ptr.patch
-- Using source at C:/Users/Public/src/vcpkg.microsoft/buildtrees/breakpad/src/2022.07.12-b097ac1ad8.clean
-- Found external ninja('1.11.0').
-- Configuring x64-windows-static-md
-- Building x64-windows-static-md-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: C:/Users/Public/src/vcpkg.microsoft/downloads/tools/cmake-3.25.1-windows/cmake-3.25.1-windows-i386/bin/cmake.exe --build . --config Debug --target install -- -v -j7
    Working Directory: C:/Users/Public/src/vcpkg.microsoft/buildtrees/breakpad/x64-windows-static-md-dbg
    See logs for more information:
      C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\install-x64-windows-static-md-dbg-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/breakpad/portfile.cmake:38 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\install-x64-windows-static-md-dbg-out.log</summary>

```
[1/69] "C:\PROGRA~1\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\cl.exe"   /TP -DNOMINMAX -DUNICODE -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -IC:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src -I"C:\Program Files\Microsoft Visual Studio\2022\Community\DIA SDK\include" /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\libbreakpad.dir\src\processor\disassembler_x86.cc.obj /FdCMakeFiles\libbreakpad.dir\libbreakpad.pdb /FS -c C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\processor\disassembler_x86.cc
[2/69] "C:\PROGRA~1\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\cl.exe"   /TP -DNOMINMAX -DUNICODE -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -IC:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src -I"C:\Program Files\Microsoft Visual Studio\2022\Community\DIA SDK\include" /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\libbreakpad.dir\src\processor\convert_old_arm64_context.cc.obj /FdCMakeFiles\libbreakpad.dir\libbreakpad.pdb /FS -c C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\processor\convert_old_arm64_context.cc
[3/69] "C:\PROGRA~1\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\cl.exe"   /TP -DNOMINMAX -DUNICODE -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -IC:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src -I"C:\Program Files\Microsoft Visual Studio\2022\Community\DIA SDK\include" /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\libbreakpad.dir\src\processor\dump_object.cc.obj /FdCMakeFiles\libbreakpad.dir\libbreakpad.pdb /FS -c C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\processor\dump_object.cc
[4/69] "C:\PROGRA~1\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\cl.exe"   /TP -DNOMINMAX -DUNICODE -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -IC:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src -I"C:\Program Files\Microsoft Visual Studio\2022\Community\DIA SDK\include" /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\libbreakpad.dir\src\processor\call_stack.cc.obj /FdCMakeFiles\libbreakpad.dir\libbreakpad.pdb /FS -c C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\processor\call_stack.cc
...
Skipped 490 lines
...

C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\processor/postfix_evaluator-inl.h(235): note: while compiling class template member function 'bool google_breakpad::PostfixEvaluator<uint32_t>::Evaluate(const std::string &,google_breakpad::PostfixEvaluator<uint32_t>::DictionaryValidityType *)'
C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\processor\stackwalker_x86.cc(386): note: see reference to function template instantiation 'bool google_breakpad::PostfixEvaluator<uint32_t>::Evaluate(const std::string &,google_breakpad::PostfixEvaluator<uint32_t>::DictionaryValidityType *)' being compiled
C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\processor\stackwalker_x86.cc(215): note: see reference to class template instantiation 'google_breakpad::PostfixEvaluator<uint32_t>' being compiled
[41/69] "C:\PROGRA~1\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\cl.exe"   /TP -DNOMINMAX -DUNICODE -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -I"C:\Program Files\Microsoft Visual Studio\2022\Community\DIA SDK\include" -IC:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\libbreakpad_client.dir\src\client\windows\crash_generation\crash_generation_client.cc.obj /FdCMakeFiles\libbreakpad_client.dir\libbreakpad_client.pdb /FS -c C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\client\windows\crash_generation\crash_generation_client.cc
[42/69] "C:\PROGRA~1\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\cl.exe"   /TP -DNOMINMAX -DUNICODE -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -I"C:\Program Files\Microsoft Visual Studio\2022\Community\DIA SDK\include" -IC:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\libbreakpad_client.dir\src\client\windows\crash_generation\client_info.cc.obj /FdCMakeFiles\libbreakpad_client.dir\libbreakpad_client.pdb /FS -c C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\client\windows\crash_generation\client_info.cc
[43/69] "C:\PROGRA~1\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\cl.exe"   /TP -DNOMINMAX -DUNICODE -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -I"C:\Program Files\Microsoft Visual Studio\2022\Community\DIA SDK\include" -IC:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\libbreakpad_client.dir\src\client\windows\crash_generation\crash_generation_server.cc.obj /FdCMakeFiles\libbreakpad_client.dir\libbreakpad_client.pdb /FS -c C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\client\windows\crash_generation\crash_generation_server.cc
[44/69] "C:\PROGRA~1\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\cl.exe"   /TP -DNOMINMAX -DUNICODE -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -I"C:\Program Files\Microsoft Visual Studio\2022\Community\DIA SDK\include" -IC:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\libbreakpad_client.dir\src\common\windows\dia_util.cc.obj /FdCMakeFiles\libbreakpad_client.dir\libbreakpad_client.pdb /FS -c C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\common\windows\dia_util.cc
FAILED: CMakeFiles/libbreakpad_client.dir/src/common/windows/dia_util.cc.obj 
"C:\PROGRA~1\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\cl.exe"   /TP -DNOMINMAX -DUNICODE -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -I"C:\Program Files\Microsoft Visual Studio\2022\Community\DIA SDK\include" -IC:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\libbreakpad_client.dir\src\common\windows\dia_util.cc.obj /FdCMakeFiles\libbreakpad_client.dir\libbreakpad_client.pdb /FS -c C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\common\windows\dia_util.cc
C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\common\windows\dia_util.cc(31): fatal error C1083: Cannot open include file: 'atlbase.h': No such file or directory
[45/69] cmd.exe /C "cd . && "C:\PROGRA~1\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\lib.exe"  /machine:x64 /nologo /out:libbreakpadd.lib CMakeFiles\libbreakpad.dir\src\processor\basic_code_modules.cc.obj CMakeFiles\libbreakpad.dir\src\processor\basic_source_line_resolver.cc.obj CMakeFiles\libbreakpad.dir\src\processor\call_stack.cc.obj CMakeFiles\libbreakpad.dir\src\processor\cfi_frame_info.cc.obj CMakeFiles\libbreakpad.dir\src\processor\convert_old_arm64_context.cc.obj CMakeFiles\libbreakpad.dir\src\processor\disassembler_x86.cc.obj CMakeFiles\libbreakpad.dir\src\processor\dump_context.cc.obj CMakeFiles\libbreakpad.dir\src\processor\dump_object.cc.obj CMakeFiles\libbreakpad.dir\src\processor\exploitability.cc.obj CMakeFiles\libbreakpad.dir\src\processor\exploitability_linux.cc.obj CMakeFiles\libbreakpad.dir\src\processor\exploitability_win.cc.obj CMakeFiles\libbreakpad.dir\src\processor\fast_source_line_resolver.cc.obj CMakeFiles\libbreakpad.dir\src\processor\logging.cc.obj CMakeFiles\libbreakpad.dir\src\processor\microdump.cc.obj CMakeFiles\libbreakpad.dir\src\processor\microdump_processor.cc.obj CMakeFiles\libbreakpad.dir\src\processor\minidump.cc.obj CMakeFiles\libbreakpad.dir\src\processor\minidump_processor.cc.obj CMakeFiles\libbreakpad.dir\src\processor\module_comparer.cc.obj CMakeFiles\libbreakpad.dir\src\processor\module_serializer.cc.obj CMakeFiles\libbreakpad.dir\src\processor\pathname_stripper.cc.obj CMakeFiles\libbreakpad.dir\src\processor\proc_maps_linux.cc.obj CMakeFiles\libbreakpad.dir\src\processor\process_state.cc.obj CMakeFiles\libbreakpad.dir\src\processor\simple_symbol_supplier.cc.obj CMakeFiles\libbreakpad.dir\src\processor\source_line_resolver_base.cc.obj CMakeFiles\libbreakpad.dir\src\processor\stack_frame_cpu.cc.obj CMakeFiles\libbreakpad.dir\src\processor\stack_frame_symbolizer.cc.obj CMakeFiles\libbreakpad.dir\src\processor\stackwalk_common.cc.obj CMakeFiles\libbreakpad.dir\src\processor\stackwalker.cc.obj CMakeFiles\libbreakpad.dir\src\processor\stackwalker_address_list.cc.obj CMakeFiles\libbreakpad.dir\src\processor\stackwalker_amd64.cc.obj CMakeFiles\libbreakpad.dir\src\processor\stackwalker_arm.cc.obj CMakeFiles\libbreakpad.dir\src\processor\stackwalker_arm64.cc.obj CMakeFiles\libbreakpad.dir\src\processor\stackwalker_mips.cc.obj CMakeFiles\libbreakpad.dir\src\processor\stackwalker_ppc.cc.obj CMakeFiles\libbreakpad.dir\src\processor\stackwalker_ppc64.cc.obj CMakeFiles\libbreakpad.dir\src\processor\stackwalker_selftest.cc.obj CMakeFiles\libbreakpad.dir\src\processor\stackwalker_sparc.cc.obj CMakeFiles\libbreakpad.dir\src\processor\stackwalker_x86.cc.obj CMakeFiles\libbreakpad.dir\src\processor\symbolic_constants_win.cc.obj CMakeFiles\libbreakpad.dir\src\processor\tokenize.cc.obj  && cd ."
[46/69] "C:\PROGRA~1\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\cl.exe"   /TP -DNOMINMAX -DUNICODE -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -I"C:\Program Files\Microsoft Visual Studio\2022\Community\DIA SDK\include" -IC:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\libbreakpad_client.dir\src\client\windows\crash_generation\minidump_generator.cc.obj /FdCMakeFiles\libbreakpad_client.dir\libbreakpad_client.pdb /FS -c C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\client\windows\crash_generation\minidump_generator.cc
[47/69] "C:\PROGRA~1\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\cl.exe"   /TP -DNOMINMAX -DUNICODE -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -I"C:\Program Files\Microsoft Visual Studio\2022\Community\DIA SDK\include" -IC:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\libbreakpad_client.dir\src\common\windows\guid_string.cc.obj /FdCMakeFiles\libbreakpad_client.dir\libbreakpad_client.pdb /FS -c C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\common\windows\guid_string.cc
[48/69] "C:\PROGRA~1\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\cl.exe"   /TP -DNOMINMAX -DUNICODE -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -I"C:\Program Files\Microsoft Visual Studio\2022\Community\DIA SDK\include" -IC:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\libbreakpad_client.dir\src\client\windows\handler\exception_handler.cc.obj /FdCMakeFiles\libbreakpad_client.dir\libbreakpad_client.pdb /FS -c C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\client\windows\handler\exception_handler.cc
[49/69] "C:\PROGRA~1\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\cl.exe"   /TP -DNOMINMAX -DUNICODE -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -I"C:\Program Files\Microsoft Visual Studio\2022\Community\DIA SDK\include" -IC:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\libbreakpad_client.dir\src\common\windows\omap.cc.obj /FdCMakeFiles\libbreakpad_client.dir\libbreakpad_client.pdb /FS -c C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\common\windows\omap.cc
FAILED: CMakeFiles/libbreakpad_client.dir/src/common/windows/omap.cc.obj 
"C:\PROGRA~1\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\cl.exe"   /TP -DNOMINMAX -DUNICODE -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -I"C:\Program Files\Microsoft Visual Studio\2022\Community\DIA SDK\include" -IC:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\libbreakpad_client.dir\src\common\windows\omap.cc.obj /FdCMakeFiles\libbreakpad_client.dir\libbreakpad_client.pdb /FS -c C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\common\windows\omap.cc
C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\common\windows\omap.cc(105): fatal error C1083: Cannot open include file: 'atlbase.h': No such file or directory
[50/69] "C:\PROGRA~1\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\cl.exe"   /TP -DNOMINMAX -DUNICODE -DWIN32_LEAN_AND_MEAN -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -I"C:\Program Files\Microsoft Visual Studio\2022\Community\DIA SDK\include" -IC:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\libbreakpad_client.dir\src\common\windows\http_upload.cc.obj /FdCMakeFiles\libbreakpad_client.dir\libbreakpad_client.pdb /FS -c C:\Users\Public\src\vcpkg.microsoft\buildtrees\breakpad\src\2022.07.12-b097ac1ad8.clean\src\common\windows\http_upload.cc
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

building breakpad requires ATL, can vcpkg port handle the missing dependency gracefully (even if it means *failing* gracefully)?

**Comment by dg0yt** [comment]
> building breakpad requires ATL, can vcpkg port handle the missing dependency gracefully (even if it means failing gracefully)?

The ATL dependency should be expressed with a dependency on port `atlmfc` for platform `windows`.
It will handle the "failing gracefully".

**Comment by StarGate-One** [comment]
I agree with @dg0yt - also make sure or add the highlighted components are installed in the `Desktop Development with C++` workload of whichever version of Visual Studio you are using - my example is from VS2022:
 
![image](https://user-images.githubusercontent.com/68293825/217647306-045a6b28-0e89-415d-b126-8d445e4c60ee.png)

**Comment by JackBoosY** [comment]
related: https://github.com/microsoft/vcpkg-tool/pull/314

**Comment by JonLiu1993** [comment]
@ghesketh, does this problem still exist?

**Comment by dg0yt** [comment]
@JonLiu1993 The port bug must be fixed: https://github.com/microsoft/vcpkg/issues/29531#issuecomment-1423215632

**Comment by JonLiu1993** [comment]
> @JonLiu1993 The port bug must be fixed: [#29531 (comment)](https://github.com/microsoft/vcpkg/issues/29531#issuecomment-1423215632)

OK, I'll get to it now.
